docs/changes/v5.2.0/API-Changes.adoc

```diff
@@ -0,0 +1,6 @@
+= API Changes =
+
+== Deprecate LDAPv2 ==
+
+The `netscape.ldap.LDAPv2` have been deprecated.
```

Comment:
This sentence doesn't read well, perhaps:
The `netscape.ldap.LDAPv2` interface has been deprecated.
?

Reply:
Ah yes, fixed.